src/main/java/com/amihaiemil/docker/RtDocker.java

```diff
@@ -129,6 +129,15 @@ public DockerSystem system() {
         );
     }
 
+
+    @Override
+    public Plugins plugins() {
+        throw new UnsupportedOperationException(
+            "Plugins API is not yet implemented. If you can contribute please,"
```

Comment:
@bkuzmic Let's avoid to make string concatenations with the `+` operator; here you can use `String.join()`. Besides, this comma after "please" could be removed.